tests/Calculator/CalculatorTest.php

```diff
@@ -84,6 +84,10 @@ public function testRound()
         $this->assertEquals('2', $calculator->round(2, Money::ROUND_HALF_EVEN));
         $this->assertEquals('2', $calculator->round(2, Money::ROUND_HALF_ODD));
         $this->assertEquals('12', $calculator->round('12.50', Money::ROUND_HALF_DOWN));
+        $this->assertEquals('-2', $calculator->round(-1.5, Money::ROUND_HALF_UP));
+        $this->assertEquals('-8329', $calculator->round(-8328.578947368, Money::ROUND_HALF_UP));
+        $this->assertEquals('-8329', $calculator->round(-8328.5, Money::ROUND_HALF_UP));
```

Comment:
Well, I think the logical way of rounding is the way how @pamil described rounding, but I never realized the difference here. For consistency, I think it would be better to maintain the current behavior, as the change would cause as much confusion as the misleading itself. Keyword: documentation.

As compromise, we could have ROUND_HALF_*_NORMAL, or something like you suggested @frederikbosch, but a little bit more concise.